open_widget_framework/open_widget_framework/widget_classes.py

```diff
@@ -80,10 +78,10 @@ def render(self, request, configuration):
             )
             + "</table>"
         )
-        return format_html(select_user_html)
+        return select_user_html
```

Comment:
@rhysyngsun I moved the call to format_html up into the WidgetClassBase so that all html is escaped even if you don't remember to put it into your custom widget class

Reply:
Actually, had to look this up myself since I haven't used `format_html` before, but according to the docs `format_html(html)` is the same as `mark_safe(hmtl)`, which doesn't actually perform any escaping on the first argument, but rather escapes subsequent positional and keyword args: https://docs.djangoproject.com/en/2.1/ref/utils/#django.utils.html.format_html

Reply:
I see. That's actually how we had it originally (using format_html to escape the input data). I just misunderstood how format html works.